jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -392,7 +392,7 @@ ecma_op_function_object_get_own_property (ecma_object_t *obj_p, /**< the functio
   }
   else if (!ecma_get_object_is_builtin (obj_p))
   {
-    prop_p = ecma_op_function_try_lazy_instantiate_property (obj_p, property_name_p);
+    prop_p = ecma_op_function_try_lazy_instantiate_property ((ecma_object_t *) obj_p, property_name_p);
```

Comment:
You cannot avoid this, since reading a property (length for example) might also trigger its construction. Hence the object is extended, and an out-of-memory error can stop the engine.